### DISCUSSION TITLE
521

### DISCUSSION THREAD

**Discussion by CECSpecialistI** [crystalyragui/MARC2RDA#349] [General]
lake44me commented [on Mar 16](https://github.com/uwlib-cams/MARC2RDA/issues/198#issuecomment-1069374759)[](https://github.com/lake44me)
I am selecting "has intended audience of expression" as the property for mapping the string in $a . Reasoning is to assume target audience will not vary between expression and manifestation since content is same, so choose this over "has intended audience of manifestation".

@[lake44me](https://github.com/lake44me)
Collaborator
lake44me commented [on Mar 16](https://github.com/uwlib-cams/MARC2RDA/issues/198#issuecomment-1069387249) • 
Issue related to first indicator: definition "Display constant". The effect of this is to append the constant in front of the contents of $a in a display interface. For end users viewing descriptions, the bare content of $a (example, if indicator 1 i= 0, the constant is "Reading grade level", and the content of a $a = "3.1") wouldn't make sense without the constant being visible before the $a text Without a more granular RDA property for, say, "intended audience reading grade level of expression", or a way to use an RDA property that would modify the statement containing $a string, to include a display label value, without reifying the statement (is there?), the alternative would be to map (in a transformation) the textual value of indicator 1 (if there is one), add a colon, and concatenate this to the beginning of the content of $a as the object of the "has intended audience of expression" triple.

An obstacle to doing this might be that $a is repeatable (drat!). However, all subfield $a's in a tag would share the same indicator 1 value, so the "display constant" could be appended in front of each one as it was mapped to RDA without ambiguity. @CECSpecialistI[](https://github.com/lake44me)

@[lake44me](https://github.com/lake44me)
Collaborator
lake44me commented [on Mar 16](https://github.com/uwlib-cams/MARC2RDA/issues/198#issuecomment-1069416107) • 
The other challenge for 521 is what to do about $b, Source, defined as "Name or abbreviation of the agency or entity that determined the target audience of the item. ". Examples in MARC21 include "LENOCA" (haven't been able to find out what this is an acronym for), "Follett Library Book Co." which has groupings of grade levels, and "Center for Disabilities" (whose?)... I haven't looked but I suspect there is no MARC code or term list for $b values.

I think in order to associate the source with the content of $a, we would have to reify the statement built from $a, and then apply the Data Provenance aspect of RDA https://access.rdatoolkit.org/Guidance/Index?externalId=en-US_ala-cfa18e03-17f2-378a-874c-86515bf7e0ac . The section that seems most applicable, to me at first reading, is
CONDITION

A potential source of metadata is not a manifestation that is being described.

A potential source of metadata does not provide additional content for a metadata work.
OPTION

Record the source of metadata as a Work: [note on metadata work](https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-3decef2d-b430-3210-9e6d-c66fbe8e514d)[](https://access.rdatoolkit.org/Guidance/Index?externalId=en-US_ala-cfa18e03-17f2-378a-874c-86515bf7e0ac#)[](https://github.com/lake44me) of the metadata work and indicate that additional information is not found in the source.

Not entirely comfortable with this. I suspect a lot of this metadata, especially reading level, is taken from information on the cover/jacket/initial pages of books and the scheme is transcribed from what's found there.

Fortunately, $b is not repeatable, so only one source applies to multiple $a subfields if they are present .

@[lake44me](https://github.com/lake44me)
Collaborator
lake44me commented [on Mar 16](https://github.com/uwlib-cams/MARC2RDA/issues/198#issuecomment-1069450098)
Finally, mapping of $3 (material specified) raises similar questions to what we discussed today, in that, if there are not RDA entities (with URIs) for the part(s) of the expression or manifestation referenced in $3, we can't hope to match up that statement to more information about that part. Since $3 text isn't controlled, that hope is a faint one anyway.

Options include,

reifying the metadata "work" (triple) containing the $a statement and then adding a "note on metadata work" about the triple, or
appending the content of $3 to the end of the statement built from $a with some contextual punctuation. Something like:
[Expression URI] http://rdaregistry.info/Elements/e/P20322 "Reading grade level: 3.1 (applies to: booklet)"

would be a "lossy" method, but probably not too much of a loss. I'm ok selecting this and documenting the necessary transformations for now, if we can label this "Loss" and review at the end.
Is this approach in line with other mappings we've done?
@CECSpecialistI[](https://github.com/lake44me)

@[lake44me](https://github.com/lake44me)
Collaborator
lake44me commented [on Mar 16](https://github.com/uwlib-cams/MARC2RDA/issues/198#issuecomment-1069475645) • 
One more thought - MARC21 https://www.loc.gov/marc/bibliographic/bd521.html states a relationship with an 008 code "Information about the target audience and/or intellectual level may also be recorded in coded form in field 008/22 (Target audience) for some types of material." There is a controlled value list for this I think: https://id.loc.gov/vocabulary/maudience.html[](https://github.com/CECSpecialistI) . Whatever we do with this tag and that 008 element should probably map to RDA in a similar way.

@[CECSpecialistI](https://github.com/CECSpecialistI)
Member
Author
CECSpecialistI commented [29 days ago](https://github.com/uwlib-cams/MARC2RDA/issues/198#issuecomment-1075646247)[](https://github.com/CECSpecialistI)
I wonder whether an unconstrained property might be best here too? I'm thinking of large text or braille manifestations of the same expression being intended for different audiences? Would there ever be notes for that? Are those at the manifestation level?

@[CECSpecialistI](https://github.com/CECSpecialistI) CECSpecialistI added the [discussion needed](https://github.com/uwlib-cams/MARC2RDA/labels/discussion%20needed) label [29 days ago](https://github.com/uwlib-cams/MARC2RDA/issues/198#event-6287044220)[](https://github.com/lake44me)
@[lake44me](https://github.com/lake44me)
Collaborator
lake44me commented [29 days ago](https://github.com/uwlib-cams/MARC2RDA/issues/198#issuecomment-1075685394)
Not sure I understand your answer here. Unconstrained properties aren't supposed to be associated with RDA Entities, and we would have to have a URI for the subject of them, right?

I'm just saying, for a subfield indicating the source of a term or code in another subfield, either that gets appended to the string, or that string has to somehow become the subject of a predicate of some kind, with the source as an object. Or the triple becomes the subject (which means, it's reified, one way or another - meaning, either as a blank node which then lists the subject, predicate and object, or by creating a URI for the triple and then giving the subject, predicate and object.

Some years ago I came across [this blog post](https://blogs.library.duke.edu/dcthree/2013/07/27/the-trouble-with-triples/)[](https://github.com/CECSpecialistI) from Hugh Cayless at Duke who was working in the digital humanities area, and I think his points still hold, and I wonder if we would all be better of if his suggestion to apply an identifier (which could be made a URI) to every triple, became general practice.. ?

@[CECSpecialistI](https://github.com/CECSpecialistI) CECSpecialistI added [asynchronous discussion needed](https://github.com/uwlib-cams/MARC2RDA/labels/asynchronous%20discussion%20needed) and removed [discussion needed](https://github.com/uwlib-cams/MARC2RDA/labels/discussion%20needed) labels [28 days ago](https://github.com/uwlib-cams/MARC2RDA/issues/198#event-6295085270)[](https://github.com/pan-zhuo)
@[pan-zhuo](https://github.com/pan-zhuo)
Member
pan-zhuo commented [23 days ago](https://github.com/uwlib-cams/MARC2RDA/issues/198#issuecomment-1081384638)
The other challenge for 521 is what to do about $b, Source, defined as "Name or abbreviation of the agency or entity that determined the target audience of the item. ". Examples in MARC21 include "LENOCA" (haven't been able to find out what this is an acronym for), "Follett Library Book Co." which has groupings of grade levels, and "Center for Disabilities" (whose?)... I haven't looked but I suspect there is no MARC code or term list for $b values.

I think in order to associate the source with the content of $a, we would have to reify the statement built from $a, and then apply the Data Provenance aspect of RDA https://access.rdatoolkit.org/Guidance/Index?externalId=en-US_ala-cfa18e03-17f2-378a-874c-86515bf7e0ac . The section that seems most applicable, to me at first reading, is CONDITION

A potential source of metadata is not a manifestation that is being described.

A potential source of metadata does not provide additional content for a metadata work. OPTION

Record the source of metadata as a Work: [note on metadata work](https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-3decef2d-b430-3210-9e6d-c66fbe8e514d) of the metadata work and indicate that additional information is not found in the source.

Not entirely comfortable with this. I suspect a lot of this metadata, especially reading level, is taken from information on the cover/jacket/initial pages of books and the scheme is transcribed from what's found there.

Fortunately, $b is not repeatable, so only one source applies to multiple $a subfields if they are present .

I think this condition is applicable only when the source does not contain the information needed, like the Authority tag 675. When information is available and taken from a source that is not the book being described, the condition above this one applies (50.43.53.04):
CONDITION
A source of metadata is not a manifestation that is being described.

OPTION
Record the source of information as a [Work: source consulted](http://www.rdaregistry.info/Elements/w/#P10406) of the metadata work.

If the information is taken from within the book being described, apply the option at (18.62.79.47):
CONDITION
The source of a metadata work is a manifestation that is being described.

OPTION
Record the source of information as a [Work: recording source](http://www.rdaregistry.info/Elements/w/#P10404).

The problem with $b for me is that it records the name of an agency. It tells nothing about where that name is taken from, either the book in hand or a VES. So I'm leaning toward [Work: related agent of work](http://www.rdaregistry.info/Elements/w/#P10311) and its subtypes, or [Work: note on metadata work](http://www.rdaregistry.info/Elements/w/#P10402)[](https://github.com/pan-zhuo) stating something like 'assigned by [value of $b].'

@[pan-zhuo](https://github.com/pan-zhuo)
Member
pan-zhuo commented [23 days ago](https://github.com/uwlib-cams/MARC2RDA/issues/198#issuecomment-1081419727)
Finally, mapping of $3 (material specified) raises similar questions to what we discussed today, in that, if there are not RDA entities (with URIs) for the part(s) of the expression or manifestation referenced in $3, we can't hope to match up that statement to more information about that part. Since $3 text isn't controlled, that hope is a faint one anyway.

Options include,

reifying the metadata "work" (triple) containing the $a statement and then adding a "note on metadata work" about the triple, or
appending the content of $3 to the end of the statement built from $a with some contextual punctuation. Something like:
[Expression URI] http://rdaregistry.info/Elements/e/P20322 "Reading grade level: 3.1 (applies to: booklet)"

would be a "lossy" method, but probably not too much of a loss. I'm ok selecting this and documenting the necessary transformations for now, if we can label this "Loss" and review at the end.
Is this approach in line with other mappings we've done? @CECSpecialistI

$3 is tricky in the context of [Aggregates](https://access.rdatoolkit.org/Guidance/Index?externalId=en-US_ala-bd1f1ef9-978c-34ee-8881-acc1500130b9)[](https://github.com/lake44me). If a book is issued together with a booklet, they are two manifestations each embodying an expression. If we choose to create a manifestation URI for the whole, there are three expressions manifested:

[AggregateManifestation] rdam:P30139 [AggregatingExpression] .
[AggregateManifestation] rdam:P30139 [Expression1] . # the expression realized in the book
[AggregateManifestation] rdam:P30139 [Expression2] . # the expression realized in the booklet
[AggregatingExpression] rdae:P20319 [Expression1] .
[AggregatingExpression] rdae:P20319 [Expression2] .
I don't think we can say this:
[AggregatingExpression] rdae:P20322 "Reading grade level: 3.1 (applies to: booklet)" .
It seems that we can't reify the statement and put the qualifying info in Work: note on metadata either, because 'an aggregating expression does not incorporate or accumulate the expressions that are aggregated.' (88.69.69.51)

I think we can only say:
[Expression2] rdae:P20322 "Reading grade level: 3.1" .

But if there's another tag that describes the booklet, like:
774 08 ǂi Container of (manifestation): [information about the booklet]

Then we will need to create a manifestation URI for the booklet:
[Manifestation2] rdam:P30020 [AggregateManifestation] . # the booklet is part of the whole

It does seem hopeless that we will be able to link the booklet manifestation with the booklet expression.
A similar situation is when $3's appear in multiple tags.

An easier solution could be to use Manifestation: note on manifestation (73.81.62.04) or Expression: note on expression (39.16.08.94).

@[lake44me](https://github.com/lake44me)
Collaborator
lake44me commented [15 days ago](https://github.com/uwlib-cams/MARC2RDA/issues/198#issuecomment-1089574558)
@CECSpecialistI Hmmm. I don't see how we can create a note on expression to give details on another note on expression (which I'm choosing as the primary relation for 521 $a). They need to be linked somehow and that would involve making note number 1 be the subject of a triple, wouldn't it?

@pan-zhuo[](https://github.com/lake44me) Thanks for your comments. About $b, I was going to just append it at the end of the string with prefix of "Source". Because the definition is squishy (source agency or entity). Entity? But I'm coming around to "Assigned by".

@[lake44me](https://github.com/lake44me) lake44me added the [RSC Question](https://github.com/uwlib-cams/MARC2RDA/labels/RSC%20Question) label [15 days ago](https://github.com/uwlib-cams/MARC2RDA/issues/198#event-6375513773)[](https://github.com/lake44me)
@[lake44me](https://github.com/lake44me)
Collaborator
lake44me commented [15 days ago](https://github.com/uwlib-cams/MARC2RDA/issues/198#issuecomment-1089578220)
So I went ahead for now and have just done the unstructured mapping of everything into one prepended, appended string, labeled it "loss" and maybe we should add to the RSC questions for now? Or does anyone have more suggestions?
Again, I've mapped or deleted every row except for subfields 6 and 8. Should I move this into Awaiting Review yet? @CECSpecialistI[](https://github.com/CECSpecialistI) you could always kick it back to "In Progress" :-)

@[CECSpecialistI](https://github.com/CECSpecialistI)
Member
Author
CECSpecialistI commented [13 days ago](https://github.com/uwlib-cams/MARC2RDA/issues/198#issuecomment-1091876768)
Thank you @lake44me[](https://github.com/CECSpecialistI) , sorry for not being as responsive as I'd like to be on this discussion so far. The choices you've made here make perfect sense to me, and if you feel good about them too, leaving status "loss" or "?" on things you think need extra attention is totally fine to do as you move this to "Awaiting review".

@[CECSpecialistI](https://github.com/CECSpecialistI) CECSpecialistI added the [meeting discussion needed](https://github.com/uwlib-cams/MARC2RDA/labels/meeting%20discussion%20needed) label [2 hours ago](https://github.com/uwlib-cams/MARC2RDA/issues/198#event-6464432701)

**Comment by pan-zhuo** [comment]
Can we apply the option at [Recording a scope for validity of metadata](https://access.rdatoolkit.org/en-US_ala-cfa18e03-17f2-378a-874c-86515bf7e0ac/section_kjz_vtd_ydb) for $3?

<[Expression URI] rdae:P20322 "Reading grade level: 3.1"> rdaw:P10403 "booklet" .

And [Work: related agent of work](http://www.rdaregistry.info/Elements/w/#P10311) or a subtype for $b?

<[Expression URI] rdae:P20322 "Reading grade level: 3.1"> rdaw:P10311 "Follett Library Book Co." .

**Reply by CECSpecialistI** [reply]
That's an idea. I haven't learned much about "scope for validity of metadata" but this seems to be accurate to me. Reification is going to come up a lot in this mapping, I fear. We should probably add a method for recording it consistently in transformation notes. I like the way Zhuo has done it here, same as the way Gordon did it earlier.

**Reply by gerontakos** [reply]
I'm confused. I'll list 3 areas below. Apologies if I'm a few steps behind everyone.
(1) 521
After our meeting, should we entertain 521 as a note property we'd shuffle off to RDA manifestation properties?
Then concatenate the subfields using an SES or natural language? 
Here's a natural language example:
<manifestation> <hasIntendedAudience> "Reading grade level 3.1 (according to Follett Library Book Co.). Applies to booklet." .
(2) reification in the mapping
I'll start a new discussion on this
I don't favor the representation we're using. That is, < <triple> triple>. 
In general I don't favor recommending any reification.
(3) Scope of validity.
I would like to make decisions like this based on general principles.
So far we've had 2:
(A) Recommend output (from MARC input) of the best RDA output possible. (I believe this roughly corresponds to what Gordon meant when he said a mapping should avoid errors.)
(B) Recommend outputting RDA that minimizes loss of values.
Sometimes we have to choose one over the other. I would prioritize (A).
In the current case (scope of validity) I would like to add a refinement to "best RDA possible." Something about keeping it as simple as possible; don't add layers of complexity. Keep it "flat." Make direct statements only, like with Wikidata's "truthy" statements -- the ones everybody likes to query (in the wdt namespace).
Another general idea on why we shouldn't use RDA instruments like scope of validity: we just want to create RDA data using legacy data so that some resources can be found/identified/selected/obtained/explored. We don't need to recommend complicated, fully-developed RDA (e.g. with reification, provenance, datatype and object properties, etc)  which will be most accurately and successfully created in native RDA environments. So I recommend against scope of validity (which requires reification).

**Reply by pan-zhuo** [reply]
Yes we were discussing mapping 521 to a note on manifestation. I just copied the statement in the original post without thinking much about it.

But $3's appear in 3XX where many of the data values are structured descriptions. If we are not going to reify, then maybe these options:
1. Disregard.
2. Append a note to a controlled term.
3. Create URIs for part of manifestations, aggregated expressions/works, and use them as subjects in triples.
I'm hesitant to take any of these approaches.

**Reply by gerontakos** [reply]
321$3 values are structured? By what? I didn't know.
Not 1 Disregard. I'm not sure what you mean by 2 Append a note. 3 Create URIs I prefer to avoid.
How about something like:
521 2# $aK-3.$bFollett Library Book Co. $3 Workbook
rdam:hasIntendedAudience "K-3 (according to Follet Library Book Co.). Applies to: Workbook."
that is:
rdam:hasIntendedAudience { <<$a value>> ("according to" <<$b value>>)". Applies to: " <<$3 value>>. }
It's possble that won't hold up for all subfield values -- but it might.

**Comment by pan-zhuo** [comment]
Sorry I don't mean to hijack the discussion of 521. I was thinking about if we want to map $3 consistently across MARC fields.

$3's appear in a lot of the 3XX fields.

336	|a text |b txt |2 rdacontent
336	|3 CD |a text |b txt |2 rdacontent
337	|a unmediated |b n |2 rdamedia
337	|3 CD |a computer |b c |2 rdamedia
338	|a volume |b nc |2 rdacarrier
338	|3 CD |a computer disc |b cd |2 rdacarrier

I'm not sure appending anything to a controlled term/identifier is a good idea, even with a possible string encoding scheme.

If there are $0's/$1's or we map a controlled term/identifier to an IRI, RDA forbids recording provenance for IRIs. Then we have to create subject URIs for $3's?

Ignoring $3's is an option because not every cataloger records them, and we have inadequate MARC to start with.

**Reply by gerontakos** [reply]
I think we need a discussion dedicated to $3 that summarizes all that's been said here about that subfield.

**Comment by pan-zhuo** [comment]
What are we going to do with LC-PCC practice of not recording the element 'intended audience of manifestation?'

More generally, are we going to make our data compatible with LC-PCC PS and MGD?

**Reply by pan-zhuo** [reply]
https://access.rdatoolkit.org/PolicyStatement/Index?externalId=en-US_ala-4aceffa0-cae7-31d0-a9fa-b9afe26135aa#division_rdaId_2c7b20a9-72fc-4c39-bcb4-7c1326769930

**Comment by gerontakos** [comment]
My understanding is that entering MARC 521 info into Expression descriptions sets is not feasible. We can discuss at the next meeting, but if that decision holds, rather than lose the data, we're creating a Manifestation note. This will not improve our indexes (in this case, an index of mostly unstructured values representing different intended audiences) but it will be a potential useful piece on information for finding, identifying, etc. Your question asks if we're going to comply with LC-PCC PS's; where we are at present, the answer is: when it's convenient. I'm okay with that because our situation is unusual (transforming legacy data). If we were creating original RDA, we would still create our own application profiles and aspire to make our data compatible with LC-PCC PS's, right? However, when creating original RDA, I'm sure we would follow LC-PCC much more.

**Reply by lake44me** [reply]
I was going to interpret Gordon's point to relate to Intended audience of the manifestation being "looser semantics" and less likely to be incorrect with respect to a particular conversion where the content of the field pertains only to the manifestation, so ok with using that.  In fact I started to re-do the mapping, but it is complicated and I ran into a technical problem in Excel so haven't done so yet.

Thanks pan-zhuo, yes I think we need to pay attention to the PCC Policy Statements.

Theo, I don't see a technical reason that makes use of P20322	for 521 not feasible.

**Comment by CECSpecialistI** [comment]
# Meeting Notes 2022-04-27
RDA has multiple elements for audience. Which do we want to use? How to handle $b and $3?
What about PCC Policy Statements and MGD? These can always change. We should incorporate usages into mappings but map to most correct RDA rather than preference of PCC (when the two are different). We definitely need to add them to references and take them into consideration.
GD: audience for manifestation element comes from LRM. Meant for accessibility stuff/carrier types. RSC soft-deprecated specific notes, such as "has note on audience of manifestation" in favor of using general notes ("has note on manifestation"). Can add boilerplate language to note values for differentiated MARC notes. Can use indicators in some cases to decide WEMI level of note. Potentially use MARC Report to extract contents of fields/subfields/various indicators to check out values when making decisions. Note fields are all the same from data processing point of view, unrealistic to have high semantic expectations of note fields. Open Metadata Registry MARC vocabulary project exists (about 10 years out of date). Shows how we can build up boilerplate content from MARC21 using captions taken from MARC manual etc.
Subfields a, b, and 3 will all be one concatenated value of the same property (note on manifestation/expression depending on indicator 1 value), with "according to:" and "applies to" language.
This avoids the problem of multiple parts of the same statement floating around separately, getting jumbled between repeated fields. This problem is common to many MARC fields
We don't have any school/public librarians in this group--521 content could be especially important to them.

**Reply by CECSpecialistI** [reply]
@lake44me @GordonDunsire I was just trying to look up the property for "has note on intended audience of manifestation"/same for expression and it doesn't seem to exist in the RDA Registry or Toolkit. Is that right? 

In that case, should we be using ["has note on manifestation"](http://rdaregistry.info/Elements/m/P30137) here, or ["has intended audience of manifestation"](http://rdaregistry.info/Elements/m/P30305) (which allows for unstructured values but doesn't have a broader value of "has note on manifestation")/["has intended audience of expression"](http://rdaregistry.info/Elements/m/P30305)? Our discussion today would still be valuable/inform our choices on specific notes under "has note on manifestation/expression/etc." but maybe this doesn't fit as well into that category as we assumed?

**Comment by GordonDunsire** [comment]
There is no note on intended audience for manifestation or expression.

521 can be safely mapped to two RDA elements:

["has intended audience of expression"](http://rdaregistry.info/Elements/m/P30305) (unstructured description)
["has note on manifestation"](http://rdaregistry.info/Elements/m/P30137).

The choice is dependent on the first indicator. It seems reasonable that first indicator "0" “Reading grade level” indicates semantics that are entirely at expression level, but the other first indicators are too fuzzy to make that call, and it is safer to assume semantics at the manifestation level.

I do not think it is appropriate to use ["has intended audience of manifestation"](http://rdaregistry.info/Elements/m/P30305) (unstructured description) because the semantics imply intention (by the publisher or producer), and the example data indicate that the values are cataloguer judgement about the audience, not the intention.

This will not be impacted by any future deprecation by RSC of differentiated note elements. An unstructured description for a specific element is semantically equivalent to a differentiated note; this is one of the main reasons why specific differentiated notes are soft-deprecated. This approach also resolves the problem of associating a differentiated note with its 'proper' element for display.

So perhaps a general alternative to mapping differentiated notes from M21 to RDA is to consider unstructured descriptions of the associated element, if the semantics of the associated element allow.

**Reply by pan-zhuo** [reply]
I'm wondering when the first indicator suggests an intended audience of expression, does the presence of $3 indicate an [entity boundary](https://access.rdatoolkit.org/en-US_ala-09a3282e-2746-3454-88ed-71935da637f9/div_cdp_yv2_3mb) and thus an aggregate, and we need to bypass the aggregating expression and map to '[intended audience of representative expression](https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-d69eefb5-d2bd-3dba-8faf-4090bb26ebf7).'

**Reply by GordonDunsire** [reply]
@pan-zhuo: that's a good point. I think it is safe to assume that what is indicated in $3, for any tag that is completely associated with 'content' in its definition or scope, is one or more expressions differentiated from the whole aggregate.

For 521 0#, we then invoke [this option in the Toolkit](https://access.rdatoolkit.org/en-US_ala-d69eefb5-d2bd-3dba-8faf-4090bb26ebf7/div_xpd_psf_3lb).
